src/main/java/com/volasoftware/tinder/service/EmailContentServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.volasoftware.tinder.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+@Service
+public class EmailContentServiceImpl implements EmailContentService{
+
+    private final ResourceLoader resourceLoader;
+
+    public EmailContentServiceImpl(ResourceLoader resourceLoader) {
+        this.resourceLoader = resourceLoader;
+    }
+
+    public String createContent(String replacement, String templateClassPath) throws IOException {
+        Resource emailResource = resourceLoader.getResource(templateClassPath);
+        File emailFile = emailResource.getFile();
+        Path path = Path.of(emailFile.getPath());
+        String emailContent = Files.readString(path);
+
+        return emailContent.replace("{{content}}", replacement);
```

Comment:
Be careful to not have word content on other places! Probably is good to pick something more unique as phrase for replacement. You can use signs as well.

Reply:
is target better?

Reply:
%%%target%%% is better. 

Also move the string as a constant in Java code for reusability